lib/cisco_node_utils/bridge_domain.rb

```diff
@@ -121,6 +121,30 @@ def destroy
     #                      PROPERTIES                      #
     ########################################################
 
+    # Bridge-Domain name assigning case
+    # bridge-domain 100
+    #   name PepsiCo
+    def bd_name
+      res = config_get('bridge_domain', 'bd_name', bd: @bd_ids)
+      return default_bd_name unless res
+      res.compact.join
```

Comment:
Cannot just return nil for bd_name if its not initialized, because there is a default bd_name assigned to it when bd is created by the system and it is Bridge-Domain<id>. This case is similar to the vlan one where default vlan name is returned if there is no name in the running config.